ios/FluentUI/Vnext/List/ListCell.swift

```diff
@@ -223,6 +223,14 @@ class MSFListCellStateImpl: NSObject, ObservableObject, Identifiable, ControlCon
         return children.count
     }
 
+    var trailingItem: MSFListAccessoryType {
+        if isSelected {
```

Comment:
nit: can you use guard !isSelected to return the .checkmark and remove the else block by just leaving the `return accessoryType` below it?